api/v1alpha1/databasecluster_types.go

```diff
@@ -284,6 +284,8 @@ type DatabaseClusterStatus struct {
 	Size int32 `json:"size,omitempty"`
 	// Message is extra information about the cluster
 	Message string `json:"message,omitempty"`
+	// DefaultStorage is the storage used by default (psmdb only)
+	DefaultStorage string `json:"defaultStorage,omitempty"`
```

Comment:
Do we really need this? I really don't like it, it's a confusing field :sweat_smile:

Reply:
Well, me too, but how will the FE know what storage is a default one if we apply it deeply under the hood based on what is used for schedules/on-demand backups?

Reply:
Another option would be to add to the BE the psmdb dependency so it could check the psmdb cluster directly, but I would rather avoid this

Reply:
So we had a couple of offline discussions and we'll move with this approach but we might rename this to `ActiveStorage`.